monkey/infection_monkey/i_puppet/port_scan_data.py

```diff
@@ -3,11 +3,12 @@
 from pydantic import Field
 
 from common.base_models import InfectionMonkeyBaseModel
-from common.types import NetworkPort, PortStatus
+from common.types import NetworkPort, NetworkService, PortStatus
 
 
 class PortScanData(InfectionMonkeyBaseModel):
     port: NetworkPort
     status: PortStatus
     banner: Optional[str] = Field(default=None)
-    service: Optional[str] = Field(default=None)
+    service: Optional[NetworkService] = Field(default=None)
```

Comment:
I think this should be mandatory but default to `NetworkService.UNKNOWN`. Thoughts?